### PR TITLE
[1LP][RFR] Update httpd workaround for 59z

### PR DESCRIPTION
**Pull request by mshriver**
Apply the httpd workaround on all streams

### CODE REVIEW

File: cfme/utils/appliance/__init__.py [hunk 1]
```diff
@@ -482,8 +483,8 @@ def fix_httpd_issue(self, log_callback=None):
         """
                 full_cmd = """echo '{cmd}' >> {dst}""".format(cmd=exec_pre, dst=dst)
                 assert ssh.run_command(full_cmd).success
-                assert ssh.run_command('systemctl daemon-reload').success
-                assert ssh.run_command('systemctl restart httpd').success
+                self.daemon_reload.restart()
```
Comment (mshriver) [inline cfme/utils/appliance/__init__.py]: The SystemdService method verifies the RC on restart and raises an exception on failure.

Comment (izapolsk) [reply cfme/utils/appliance/__init__.py]: why restart ? it will cause error.  I'd add reload method to SystemdService class instead

Comment (mshriver) [reply cfme/utils/appliance/__init__.py]: OH sorry I totally mis-read this.

File: cfme/utils/appliance/__init__.py [hunk 2]
```diff
@@ -482,9 +482,9 @@ def fix_httpd_issue(self, log_callback=None):
         """
                 full_cmd = """echo '{cmd}' >> {dst}""".format(cmd=exec_pre, dst=dst)
                 assert ssh.run_command(full_cmd).success
-                assert ssh.run_command('systemctl daemon-reload').success
-                assert ssh.run_command('systemctl restart httpd').success
-                self.wait_for_web_ui(timeout=1800, log_callback=log_callback)
+                self.httpd.daemon_reload()
+                self.httpd.restart()
+                wait_for(lambda: self.httpd.running, timeout=1800)
```
Comment (mshriver) [inline cfme/utils/appliance/__init__.py]: This is run during configuration, where webui likely won't be up.

**Pull request merged**